templates/mainapp/404.html

```diff
@@ -0,0 +1,154 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+	<!-- Required meta tags -->
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<!-- Bootstrap CSS -->
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet"
+		integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
+
+	<title>404 page not found</title>
+</head>
+<style>
+	body {
+		background-color: #f9ce84;
+		font-family: "Sora", sans-serif;
```

Comment:
remove this

Reply:
Not possible

Reply:
why?

Reply:
I had set the background colour of same as of background colour of gif .